### PR TITLE
[DX-970] Update release-notes-template.md with optional "Changed error log mes…

### PR DESCRIPTION
**Pull request by letzya**
…sage"

### For internal users - Please add a Jira DX PR ticket to the subject!
<br>

---

### Preview Link

<br>

### Description

<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labelled** the PR

### CODE REVIEW

**Comment by letzya** [comment]
@dcs3spp FYI I noticed we need to improve the order of the headings.

File: tyk-docs/assets/others/release-notes-template.md [hunk 1]
```diff
@@ -34,13 +34,16 @@ Our minor releases are supported until our next minor comes out. This would be <
 <!-- Use the following statement if there are no breaking changes, or explain if there are -->
 This release has no breaking changes.
 
+#### Changed error log messages (Optional)
```
Comment (dcs3spp) [inline tyk-docs/assets/others/release-notes-template.md]: @letzya Agreed that should log changed error log messages in changelog of release notes template. Not sure if we should duplicate it in a new section of the release notes also? Release notes guidelines internal Confluence will also need updating following the outcome of this PR

Comment (letzya) [reply tyk-docs/assets/others/release-notes-template.md]: my thought was **to make sure the users don't miss this notice among other changelog lines**. When it's in the changelog, you need to expend it and also it's scattered along the changelog list. Here it's a clear visible list with a specific heading to draw the readers attention

Comment (letzya) [reply tyk-docs/assets/others/release-notes-template.md]: It's important for users that are using our logs for monitoring. Will add this explanation

Comment (letzya) [reply tyk-docs/assets/others/release-notes-template.md]: I updated the content with an explanation

**Review by dcs3spp — APPROVED** [review]
@letzja PR LGTM

**Pull request merged**